Minor update: Refine XML( ) example 4 (#127845)
Azure/azure-content: articles/logic-apps/expression-functions-reference.md

The `xml()` function expects either an object or a string containing valid XML. The function doesn't accept a raw array as input.

If you have a JSON array, like the following example, you have four options.

```json
[
  { "ID": 1, "Name": "James" },
  { "ID": 2, "Name": "John" },
  { "ID": 3, "Name": "Sam" }
]
```

> Option 1: Convert JSON string to a JSON object before you pass the result to the `xml()` function, for example:

Reviewer: ```suggestion
Option 1: Convert the JSON string to a JSON object before you pass the result to the `xml()` function, for example:
```

Reviewer: Hi @watana2, I don't see any changes made.

    )
  )
)
```

> Option 4: Similar to option 3, but uses `addProperty()` instead of the `concat()` function to create the JSON object before passing it to the `xml()` function, for example: 

Reviewer 2: Not applicable. OK to ignore this suggestion.

)
```

> All examples, which include options 1 to 4, return the following XML result:

Reviewer: ```suggestion
All the examples from options 1 to 4 return the following XML result:
```
